observability/src/main/java/io/grpc/observability/logging/GcpLogSink.java

```diff
@@ -78,13 +97,17 @@ public void write(GrpcLogRecord logProto) {
     try {
       GrpcLogRecord.EventType event = logProto.getEventType();
       Severity logEntrySeverity = getCloudLoggingLevel(logProto.getLogLevel());
+      MonitoredResource resource = getResource(locationTags);
       // TODO(vindhyan): make sure all (int, long) values are not displayed as double
```

Comment:
When will this be done? We would need this before asking users to use/test this, right?

Reply:
I do not see this issue with `int` values in the cloud logging UI now.
But all `long` values are being converted to String as part of `JsonFormat.printer().print`. Added comment.